docs/patterns/CVE-2018-20992-MIR.rpl

```diff
@@ -0,0 +1,19 @@
+pattern CVE-2018-20992-MIR
+
+patt {
+    use std::vec::Vec;
+
+    p_vec_set_len_to_extend[
+        $T: ty
+    ] = #[mir] pub fn _ (..) -> _ {
+        let $vec: Vec<$T> = _;
+        let $new_len: usize = _;
+        let $vec_ref: &Vec<$T> = &$vec;
+        let $old_len: usize = copy ((*$vec_ref).len);
+        // SwitchInt Here or If here?
```

Comment:
If I try to express `#without!(if condition) {}` in RPL MIR pattern, should I use `#without! if` or `#without! switchInt`?
@frank-king @TheVeryDarkness

Reply:
I think `if` can be added as a syntactic sugar for `switchInt` with a boolean operand.